lib/csv_record.rb

```diff
@@ -24,7 +24,7 @@ def self.load_all(full_path: nil, directory: nil, file_name: nil)
 
     def self.validate_id(id)
       if id.nil? || id <= 0
-        raise ArgumentError, 'ID cannot be blank or less than one.'
+          raise ArgumentError, 'ID cannot be blank or less than one.'
```

Comment:
This was right before:

```suggestion
        raise ArgumentError, 'ID cannot be blank or less than one.'
```